networkd/network_server.ml

```diff
@@ -844,6 +844,32 @@ module Bridge = struct
 				Sysfs.bridge_to_interfaces name
 		) ()
 
+	let get_physical_interfaces _ dbg ~name =
+		Debug.with_thread_associated dbg (fun () ->
+			match !backend_kind with
+			| Openvswitch ->
+				Ovs.get_real_bridge name
+				|> Ovs.bridge_to_interfaces
+				|> List.filter (Sysfs.is_physical)
+
+			| Bridge ->
+				let iface =
+					match List.filter (fun iface -> not (String.startswith "vif" iface)) (Sysfs.bridge_to_interfaces name) with
```

Comment:
You'll need a different way to determine whether an interface is a VLAN interface in the Linux kernel. This is separate from the bridge stuff (unlike for the OVS).